x/evm/keeper/grpc_query.go

```diff
@@ -421,9 +421,11 @@ func (k Keeper) TraceTx(c context.Context, req *types.QueryTraceTxRequest) (*typ
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "failed to load evm config: %s", err.Error())
 	}
-	signer := ethtypes.MakeSigner(cfg.ChainConfig, big.NewInt(ctx.BlockHeight()))
-
+	height := ctx.BlockHeight()
+	patch := height < req.FixClearAccessListHeight
```

Comment:
Here, I see that you are accessing the `FixClearAccessListHeight` from the request itself. Similar to the question above: why does the constant have to come from the request? Why can't it be stored elsewhere, ex. in the code for the evm keeper itself, to avoid changing the proto files and json-rpc configuration overall?

Reply:
Not sure any better way to access JSONRPCConfig in keeper?